spec/index.html

```diff
@@ -581,6 +609,186 @@ <h4>The <code>&lt;<dfn id="the-mapml-viewer-element">mapml-viewer</dfn>&gt;</cod
         <p>The <a href="https://drafts.csswg.org/css-images/#default-object-size">default object size</a> is a width of 300 <a href="https://drafts.csswg.org/css-values/#px">CSS pixels</a> and a height of 150 <a href="https://drafts.csswg.org/css-values/#px">CSS pixels</a>. [[css-images-3]]</p>
+        
+        <p>
+          The <a href="#dom-htmlmapmlviewerelement-back"><code>back</code></a>, 
+          <a href="#dom-htmlmapmlviewerelement-forward"><code>forward</code></a> 
+          and <a href="#dom-htmlmapmlviewerelement-reload"><code>reload</code></a> 
+          operations allow the manipulation of the <a href="#the-mapml-viewer-element"><code>mapml-viewer</code></a>'s
+          viewport history stack.  Each time the <a href="#the-mapml-viewer-element"><code>mapml-viewer</code></a>'s viewport 
+          comes to rest, its location (center coordinates and zoom level) is pushed 
+          onto the viewport history stack.  
+          The <a href="#dom-htmlmapmlviewerelement-back"><code>back</code></a> 
+          operation will go back (down the stack) one increment in the viewport 
+          history stack. 
+          The <a href="#dom-htmlmapmlviewerelement-forward"><code>forward</code></a> 
+          operation will go forward (up the stack) by one increment, if there are 
+          more recent viewport locations on the stack. The viewport history content 
+          is erased, having all its entries deleted,
+          through the <a href="#dom-htmlmapmlviewerelement-reload"><code>reload</code></a> 
+          operation. The state of the viewport history stack is reflected by the 
+          interactive or disabled state of the Forward, Back and Reload viewer 
+          context menu items, as well as by the interactive state of the viewer 
+          reload control button.
+        </p>
+        <p>
+          The <a href="#dom-htmlmapmlviewerelement-geojson2mapml"><code>geojson2mapml</code></a> 
+          operation transcribes a GeoJSON [[rfc7946]] 
+          <a href="https://www.rfc-editor.org/rfc/rfc7946#section-3.3">FeatureCollection</a> or 
+          <a href="https://www.rfc-editor.org/rfc/rfc7946#section-3.2">Feature</a> 
+          object into a  <a href="#the-layer-element"><code>layer</code></a> element. 
+
+          <a href="#dom-htmlmapmlviewerelement-geojson2mapml"><code>geojson2mapml</code></a>
+          accepts two arguments: the required JSON object, conforming to a GeoJSON [[rfc7946]]
+          <a href="https://www.rfc-editor.org/rfc/rfc7946#section-3.3">FeatureCollection</a> or 
+          <a href="https://www.rfc-editor.org/rfc/rfc7946#section-3.2">Feature</a>, 
+          and an optional dictionary object of options. 
+          <a href="#dom-htmlmapmlviewerelement-geojson2mapml"><code>geojson2mapml</code></a> 
+          creates and adds a <a href=""><code>layer</code></a> element as the last 
+          child of the <a href=""><code>mapml-viewer</code></a> element on which
```

Comment:
Empty links here also

Reply:
Thanks for the close reading, this is why a review is worth its weight in gold :-)